Admin/BaseMediaAdmin.php

```diff
@@ -169,7 +175,7 @@ public function getNewInstance()
     }
 
     /**
-     * @return null|\Sonata\MediaBundle\Provider\Pool
+     * @return Pool|null
```

Comment:
this is wrong IMO, this can't be null, because we have typehint in the ``__construct()`` method and this comes from the DI